src/XIVLauncher/Updates.cs

```diff
@@ -26,18 +28,20 @@ public async Task Run(bool downloadPrerelease, ChangelogWindow changelogWindow)
 
             try
             {
-                ReleaseEntry newRelease = null;
+                ReleaseEntry? newRelease = null;
 
                 using (var updateManager = new UpdateManager(url, "XIVLauncher"))
                 {
                     // TODO: is this allowed?
                     SquirrelAwareApp.HandleEvents(
```

Comment:
Should we still be doing this? The Clowd.Squirrel readme recommends using app tool methods instead and doing it when the app starts, not here. updateManager might also be disposed at this point.

Reply:
I moved the handling of Squirrel events to the App startup using tool methods. Squirrel log has also been added to the XIVLauncher log.